langchain/chains/conversational_retrieval/base.py

```diff
@@ -107,7 +114,13 @@ def _call(
             )
         else:
             new_question = question
-        docs = self._get_docs(new_question, inputs)
+        accepts_run_manager = (
```

Comment:
imo this may be overkill. this only matters if someone has subclassed this and overridden _get_docs right?

Reply:
It mostly applies to people who have existing retriever implementations that don't accept the run manager

Reply:
Yeah, though _get_docs is abstract so it would be anyone who subclasses this outside our repo. I don't know if it is the case